src/components/biggive-basic-card/biggive-basic-card.tsx

```diff
@@ -81,21 +81,21 @@ export class BiggiveBasicCard {
         <a href={this.buttonUrl}>
           <div class={'sleeve background-colour-' + this.cardColour + ' text-colour-' + this.textColour}>
             <div class="content-wrap">
-              {this.icon == true ? (
+              {this.icon ? (
                 <div class="icon">
                   <svg width="53" height="39" viewBox="0 0 53 39" fill="none" xmlns="http://www.w3.org/2000/svg">
                     <path d="M38.7009 13.6572L52.3535 38.6959H25.0386L38.7009 13.6572Z" class="fill-black" />
                     <path d="M20.4789 36.4199L6.36785e-06 5.89713e-05L40.9724 6.61352e-05L20.4789 36.4199Z" class={'fill-' + this.iconColour} />
                   </svg>
                 </div>
               ) : null}
-              {this.mainImageUrl != '' ? (
+              {(this.mainImageUrl?.length ?? 0) === 0 ? null : (
```

Comment:
```suggestion
              {! (this.isEmpty(this.mainImageUrl) ? null : (
```

or with the ternary logic reversed:

```typescript
              {!! this.mainImageUrl ? ( ...
```